src/utils/NovorAdapter.cpp

```diff
@@ -383,15 +387,22 @@ class TOPPNovorAdapter :
 
       // extract version from comment 
       // #              v1.06.0634 (stable)
+      // v1.06.0634 (stable)
       vector<ProteinIdentification> protein_ids;
       StringList versionrow;
       csv.getRow(2, versionrow);
-      versionrow[0].suffix('#').trim();
-        
+      String version;
+      boost::smatch match;
+      boost::regex regexp("v.*");
+      if(boost::regex_search(versionrow[0], match, regexp))
```

Comment:
suggestion: this regex can be replaced by a simple
`version = versionrow[0].substr(versionrow[0].find("v."));`